testament/testament.nim

```diff
@@ -727,7 +744,8 @@ proc main() =
     isMainProcess = false
     var cat = Category(p.key)
     p.next
-    processCategory(r, cat, p.cmdLineRest.string, testsDir, runJoinableTests = false)
+    processCategory(r, cat, p.cmdLineRest.string, testsDir,
```

Comment:
reformatting in unrelated parts of code should not be part of this PR but be done in separate PR (and only if it's via running nimpretty with default options); ditto wherever relevant.
The reason is this makes it hard to see actual vs pure reformatting changes

Reply:
I will rework this PR as a set of smaller PRs and make sure to disable the IDE/editor prettifier to avoid auto reformatting.